repos/system_upgrade/el7toel8/actors/spamassassinconfigcheck/libraries/library.py

```diff
@@ -0,0 +1,86 @@
+import os
+
+from leapp import reporting
+from leapp.libraries.common.spamassassinutils import \
+    SPAMC_CONFIG_FILE, SPAMASSASSIN_SERVICE_OVERRIDE, SYSCONFIG_SPAMASSASSIN
+
+
+def _check_spamc_config(facts, report_func):
+    title = ('spamc no longer allows specifying the TLS version and no longer '
+             'supports SSLv3')
+    summary_generic = ('spamc no longer allows using the "--ssl" option with an '
+                       'argument specifying the TLS version - the option can only '
+                       'be used without an argument. Also, spamc no longer supports '
+                       'SSLv3.')
+    if facts.spamc_ssl_argument:
+        summary_detail = ('The configuration file %s uses "--ssl %s", it will '
+                          'be updated during the upgrade.'
+                          % (SPAMC_CONFIG_FILE, facts.spamc_ssl_argument))
+        summary = reporting.Summary('%s %s' % (summary_generic, summary_detail))
+    else:
+        summary = reporting.Summary(summary_generic)
+    severity = (reporting.Severity.HIGH if facts.spamc_ssl_argument == 'sslv3'
+                else reporting.Severity.MEDIUM)
+    hint = 'Please update your scripts and configuration, if there are any.'
+    report_func([
+        reporting.Title(title),
+        summary,
+        reporting.Severity(severity),
+        reporting.Remediation(hint=hint),
+    ])
```

Comment:
Consider adding resources (file) and tags.
https://leapp.readthedocs.io/en/latest/actor-rhel7-to-rhel8.html#reporting-tips

Reply:
Done.